### DISCUSSION TITLE
Import Torch doesn't work in Bonsai RX

### DISCUSSION THREAD

**Discussion by lblabucsd** [bonsai-rx/bonsai#1567] [Q&A]
Dear Bonsai Crew,
I am struggling with **importing torch in a PythonTransform**, in order to run a model that was trained in pytorch. I believe I have installed all packages necessary (and beyond at this point :). However, as soon as typing "import torch" the PythonTransform node returns an error, saying "**No module named torch**". It would be awesome to get some input on a potential solution. I am adding the list of installed packages in a screenshot below.
Thank you so much and best regards,
Lisanne

![image](https://github.com/bonsai-rx/bonsai/assets/69403596/eccdfcfb-dc1e-426e-971e-40bc72a0e193)
![image](https://github.com/bonsai-rx/bonsai/assets/69403596/860b0fae-9a85-466b-b30e-7342e471c3af)

System info: Windows 10 Pro, x64-based processor

**Comment by bruno-f-cruz** [comment]
The IronPython package will not allow you to import Pytorch easily (or at all to be honest). If you want to use this module, I would recommend trying the new Python package. The documentation can be found [here](https://bonsai-rx.org/python-scripting), along with examples on how to run your custom python kernel.
